libraries/AP_Scripting/tests/rngfnd_quality_test.lua

```diff
@@ -0,0 +1,209 @@
+
+
+
+-- The UPDATE_PERIOD_MS is the time between when a distance is set and
+-- it is read. There is a periodic task that copies the set distance to
+-- a place it is read from. If UPDATE_PERIOD_MS is too short this periodic
+-- task might not get a chance to run. A value of 25 seems to be too quick.
+local UPDATE_PERIOD_MS = 50
```

Comment:
The update polling rate depends on vehicle type. I did a quick search for `SCHED_TASK(read_rangefinder` and it looks like sub and copter poll at 20Hz, but plane and rover poll at 50Hz. I'm not sure how to handle this.

Reply:
Interesting. The test is running in sub which updates every 50ms. That is why 25ms for the test occasionally had problems.